packages/core-flows/src/fulfillment/steps/delete-service-zones.ts

```diff
@@ -10,7 +10,7 @@ export const deleteServiceZonesStep = createStep(
       ModuleRegistrationName.FULFILLMENT
     )
 
-    await service.softDelete(ids)
+    await service.softDeleteServiceZones(ids)
```

Comment:
Ive seen this in few places :)

Reply:
False positive test because of idempotent operation. We need to be more deliberate about testing for existence separately from the deletion 😄